blog/_posts/2022-07-01-teach-programming-with-scala.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy (Scala Center)
+title: "Teach Programming with Scala"
+---
+
+Scala is more than a great programming language for building software.
+It is also a great tool for teaching programming!
+
+We are happy to announce that we have published a [new page][teach] on the
```

Comment:
Would be nice to have a "Teach" link in the main nav bar of the site.

Reply:
Maybe, but I am not sure yet, so I would rather add it later.